README.md

```diff
@@ -1,6 +1,38 @@
-# [GeoCAT Applications](https://ncar.github.io/geocat-applications/)
+# GeoCAT Applications
 
-This is the source repository for GeoCAT-applications.
+| **License**    |                                                                        [![License][license-badge]][repo-link]                                                                        |
+| :-----------   | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+
+## What is [GeoCAT Applications](https://ncar.github.io/geocat-applications/)?
+
+GeoCAT Applications is a community resource managed by the GeoCAT team. Inspired by
+[NCL Applications](https://www.ncl.ucar.edu/Applications/).
+
+GeoCAT Applications is broken into two main sections:
+1. Applications
+2. NCL to Python
+
+### Applications
+Applications is designed to be a quick reference to demonstrate capabilities within the Scientific
+Python Ecosystem that may be relevant to your geoscience workflows
```

Comment:
```suggestion
Python ecosystem that may be relevant to your geoscience workflows
```

Given that there's not really a formal definition / structure to point to, I think this probably shouldn't be capitalized.